lib/dullard/reader.rb

```diff
@@ -7,7 +7,7 @@ class Error < StandardError; end
   SharedStringPath = 'xl/sharedStrings.xml'
   StylesPath = 'xl/styles.xml'
 
-  class Time < Struct.new('Time', :hours, :minutes, :seconds)
+  class Time < Struct.new('DullTime', :hours, :minutes, :seconds)
```

Comment:
It seems the preferred way to create an anonymous struct would be
``Time = Struct.new(:hours, :minutes, :seconds)``